pkgs/development/python-modules/nipype/default.nix

```diff
@@ -36,6 +36,8 @@ buildPythonPackage rec {
     sha256 = "47f62fda3d6b9a37aa407a6b78c80e91240aa71e61191ed00da68b02839fe258";
   };
 
+  patches = [ ./prov-version.patch ];
```

Comment:
Please add a comment that links to this issue: https://github.com/nipy/nipype/issues/2240